aws/alarms/lambda/notify_slack/notify_slack.js

```diff
@@ -27,11 +27,11 @@ function getMessage(message) {
 function getSNSMessageSeverity(message) {
   const errorMessages = ["error", "critical"];
   const warningMessages = ["warning", "failure"];
-  const alarm_ok_status = '"NewStateValue":"OK"'; // This is the string that is returned when the alarm is reset
+  const alarm_ok_status = '"newstatevalue":"ok"'; // This is the string that is returned when the alarm is reset
```

Comment:
Probably another bug that we missed... The lowercase action below would have made it impossible to detect this string

Reply:
Probably why we did not see any reset notification in Slack for a while